src/modules/pricing/pricing-calculator/style.module.css

```
@@ -0,0 +1,105 @@
+.root {
+  flex: 1;
+  --gap: 2rem;
+}
+
+.card {
+  background-color: var(--theme-card-bg-color);
+  border-radius: var(--theme-card-border-radius);
+  padding: 2rem;
+  display: flex;
+  flex-direction: column;
+
+  gap: var(--gap);
+  position: relative;
+}
+
+.row {
+  display: flex;
+  flex-direction: row;
+  gap: var(--gap);
+
+  justify-content: space-around;
+  align-items: center;
+
+  flex-wrap: wrap;
+  width: 100%;
+
+  position: relative;
+}
+
+.card > .row {
+  flex-basis: 100%;
+}
+
+
+.labelledInput {
+  flex-grow: 0;
+
+  display: flex;
+  gap: 0.5rem;
+  align-items: center;
+}
+
+.labelledInput .label {
+  flex-basis: 20%;
+}
+
+.regionSelect {
+  width: 100%;
+}
+
+.sliders {
+  flex: 1;
+  position: relative;
+}
+
+.sliders .labelledInput {
+  align-items: baseline;
+}
+
+.priceCalculation {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  gap: 0.5rem;
+  min-width: 100%;
+}
+
+.priceCalculation h2 {
+  margin: unset;
+}
+
+.compressionInfo {
+  width: 100%;
+  height: 4rem;
+  color: var(--palette-pale-blue);
+}
+
+.compressionInfo .accent {
+  color: var(--theme-card-text-color);
+}
+
+@media only screen and (min-width: 600px) {
+  .card {
+    padding: 2rem 4rem;
+  }
+  .card > .row {
+    flex-basis: 50%;
+  }
+
+  .compressionInfo {
+    width: 50ch;
```

```suggestion
    width: 50ch;
```

maybe this can be expressed via percentage?

---

The main issue I'm encountering is that when the width is anything other than a fixed constant, there's a weird interplay between the length of this content and the overall width of the card as sliders move, resulting in a really disorienting jitter effect. Fixes are very welcomed if you have ideas.